tests/pre_commit_hook_test.py

```diff
@@ -68,7 +69,7 @@ def test_baseline_filters_out_known_secrets():
             ])
 
         # Remove one arbitrary secret, so that it won't be the full set.
-        secrets.data['test_data/each_secret.py'].pop()
+        secrets.data[str(Path('test_data/each_secret.py'))].pop()
```

Comment:
```
>>> str(Path('test_data/each_secret.py'))
'test_data/each_secret.py'
```
is there any benefit from casting `Path` to `str` that I'm missing? Otherwise it appears to be an unnecessary operation given that the parameter passed to `Path` is what that casting would return.